src/qml/DigitizingToolbar.qml

```diff
@@ -72,6 +72,7 @@ Row {
     bgcolor: "#FFD600"
 
     onClicked: {
+      vertexRemoved()
```

Comment:
I'm not sure about that. What is when I make:
- Start digitizing (pen-icon)
- Draw a line and press +
- Draw another line and then press the floppy (save)
Then the form opens but the last expected point (to have a triangle) is not set anymore.

Reply:
I will precise myself. Wait one moment, then I show you in pictures.

Reply:
When made (pen) draw line (+) draw line:
![screenshot_2018-08-03-14-42-38](https://user-images.githubusercontent.com/28384354/43643662-4c1d1eac-972c-11e8-8fab-5649bbef6439.png)
On the floppy here I expect it saves the third point (for a triangle)...

But this next example is improved in your version thanks to the freeze:
And here I made (pen) draw line (+) draw line (+) draw line (floppy) -> form opens:
![screenshot_2018-08-03-14-27-30](https://user-images.githubusercontent.com/28384354/43643663-4c3bdc52-972c-11e8-9a12-4a7c92081c24.png)
This is the old version. Here it saves not a square (4 corners) when I press the final floppy but a pentagon (5 corners) what is not expected. So this is much better in your version.

Reply:
indeed it needs to check when there is enough points to save, will fix it